scripts/import/import_EPMC.pl

```diff
@@ -137,9 +137,10 @@ sub import_citations{
     my $data           = shift;
     my $type           = shift;
 
-    ## store variants not from this species
     open my $error_log, ">>$species\_$type\_notfound_" . log_time() . "\.log"|| die "Failed to open log file: $!\n";
-
+    ## store variants not from this species
+    open my $not_found, ">>rsID\_$type\_notfound\.csv"|| die "Failed to open file: $!\n";
```

Comment:
Not using brackets and the lower precedence of || means that the file open is not checked